netlify.toml

```
@@ -0,0 +1,16 @@
+[build]
+  command = """
+    rustup install nightly --profile minimal && cargo doc --no-deps --features full
+    """
+  publish = "target/doc"
+
+[build.environment]
+  RUSTDOCFLAGS="""
+    --cfg docsrs \
+    --cfg tokio_unstable \
+    """
+  RUSTFLAGS="--cfg tokio_unstable"
```

This makes a difference for the documentation in tokio-stream.
```suggestion
  RUSTFLAGS="--cfg tokio_unstable --cfg docsrs"
```